be/src/common/constexpr.h

```diff
@@ -23,4 +23,7 @@ constexpr const int DEFAULT_CHUNK_SIZE = 4096;
 constexpr int NUM_LOCK_SHARD_LOG = 5;
 constexpr int NUM_LOCK_SHARD = 1 << NUM_LOCK_SHARD_LOG;
 
+// 1GB limit
```

Comment:
I suggest that you should tell what this constant expression is used for not the default value.